velox/substrait/tests/Substrait2VeloxPlanConversionTest.cpp

```diff
@@ -29,6 +29,7 @@ using namespace facebook::velox;
 using namespace facebook::velox::test;
 using namespace facebook::velox::connector::hive;
 using namespace facebook::velox::exec;
+using namespace facebook::velox::substrait;
```

Comment:
Unintended change?

Reply:
Yes, it is unintended change. I should remove it.  I was careless when I make a rebase.